schemas/spec-v2.schema.json

```diff
@@ -0,0 +1,311 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://switchboard.clarin.eu/tool-spec-v2.json",
+    "title": "JSON Schema for the Switchboard tool specification v.2",
+    "type": "object",
+    "items": null,
+    "required": [
+        "formatVersion",
+        "task",
+        "deployment",
+        "name",
+        "description",
+        "logo",
+        "homepage",
+        "creators",
+        "contact",
+        "inputs"
+    ],
+    "additionalProperties": false,
+    "properties": {
+        "formatVersion": {
+            "type": "string",
+            "enum": ["2"]
+        },
+        "task": {
```

Comment:
So it does become a free string in schema v2? [see](https://github.com/clarin-eric/switchboard-tool-registry/pull/81#issuecomment-736386849)

Reply:
I'll copy the list from v1 then.

Reply:
done in the new commit